sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -3,12 +3,11 @@
 ## 1.0.0-beta.4 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added filter to DeviceManagementOperations.listDeviceClasses method
+- Updated description for some methods to be more descriptive and less ambiguous
 
 ### Other Changes
+- Removed filter from DeviceManagementOperations.listBestUpdatesForGroup method
```

Comment:
Add dependency updates similar to previous releases:

#### Dependency Updates
- Upgraded `azure-core` from `1.30.0` to version `1.31.0`.
- Upgraded `azure-core-http-netty` from `1.12.3` to version `1.12.4`.